src/App/App.tsx

```diff
@@ -10,6 +10,7 @@ import { Nav } from '../Nav/Nav'
 import { PageNotFoundPage } from '../PageNotFoundPage/PageNotFoundPage'
 import { SearchUsersPage } from '../SearchUsersPage/SearchUsersPage'
 import './App.css'
+import { DisplayUsersPage } from './DsiplayUsersPage/DisplayUsersPage'
```

Comment:
There's a typo in the folder, so this import will need updated

```suggestion
import { DisplayUsersPage } from './DisplayUsersPage/DisplayUsersPage'
```

Reply:
Fixed